### PR TITLE
Fix `Link to "Special:Browse" no longer shown for Vector`

### PR DESCRIPTION
**Pull request by StarHeartHunt**

### CODE REVIEW

**Comment by StarHeartHunt** [comment]
Fix  #4831

**Comment by kghbln** [comment]
@JeroenDeDauw Could you have a peep...

File: src/MediaWiki/Hooks/SidebarBeforeOutput.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+use SpecialPage;
+use Title;
+use SMW\OptionsAwareTrait;
+/**
+ * Called at the end of Skin::buildSidebar().
+ *
+ * @see https://www.mediawiki.org/wiki/Manual:Hooks/SidebarBeforeOutput
+ *
+ * @license GNU GPL v2+
+ * @since 3.2
```
Comment (JeroenDeDauw) [inline src/MediaWiki/Hooks/SidebarBeforeOutput.php:18]: Please drop the since tags, they are incorrect

**Comment by JeroenDeDauw** [comment]
@StarHeartHunt did you test this? Against which MW version(s)?

It is a bit scary to merge this new code without any automated tests for it.

File: src/MediaWiki/Hooks/SidebarBeforeOutput.php (hunk 1 above)
Comment (StarHeartHunt) [reply src/MediaWiki/Hooks/SidebarBeforeOutput.php]: Dropped in the latest commit

**Comment by StarHeartHunt** [comment]
> @StarHeartHunt did you test this? Against which MW version(s)?
> 
> It is a bit scary to merge this new code without any automated tests for it.

I have tested this on [1.35](https://mw.bakacraft.net/w/%E9%A6%96%E9%A1%B5) & [1.31.10](https://test.bakacraft.net/index.php?title=%E9%A6%96%E9%A1%B5).I checked the behavior and found nothing wrong.Since I couldn't get the PHPUnit tests to run locally, I gave up to write a test script.☹️

**Comment by JeroenDeDauw** [comment]
Uhhhhh this broke the CI

> There was 1 failure:
> 
> 1) SMW\Tests\MediaWiki\HooksTest::testCheckOnMissingHandlers
> 
> Missing a `SidebarBeforeOutput` test!
> 
> Failed asserting that an array has the key 'SidebarBeforeOutput'.
> 
> /home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/tests/phpunit/Unit/MediaWiki/HooksTest.php:257
> 
> /home/travis/build/SemanticMediaWiki/mw/maintenance/doMaintenance.php:94

(from https://travis-ci.org/github/SemanticMediaWiki/SemanticMediaWiki/jobs/735989762)

@StarHeartHunt you could write it without having the tests run locally. Just push to a branch here on GitHub and see if it works on Travis.

**Comment by kghbln** [comment]
I guess this is somehow breaking for SMW 3.2.x and MW 1.31.x in conjunction with the Medik skin 3.x, i.e. I either get the MediaWiki tools in the drowdown menu OR the link to Special:Browse. However since MW 1.31.x is on the way out in two months we should no longer worry. To get at least the MediaWiki stuff working again one has to set:

 `$smwgBrowseFeatures = ( $smwgBrowseFeatures & ~SMW_BROWSE_TLINK ); `

On MW 1.35.x with SWM 3.2.x and Medik 3.2.x everything is fluff. See sandbox.s-mw.o

**Pull request merged**